i18n/locales/en.json

```
@@ -43,19 +43,19 @@
   "components.article.editLink.title.translate": "Interested to help with translations?",
   "components.common.languageSelector.button.title": "Switch Language",
   "components.blog.blogCard.author.by": "by",
-  "components.footer.links.trademark": "Trademark Policy",
-  "components.footer.links.privacy": "Privacy Policy",
-  "components.footer.links.codeOfConduct": "Code of Conduct",
-  "components.footer.links.security": "Security",
-  "components.footer.links.about": "About",
-  "components.footer.links.blog": "Blog",
-  "components.footer.links.github": "GitHub",
-  "components.prevnextlink.next": "NEXT",
-  "components.prevnextlink.previous": "PREV",
-  "pages.index.features.javascript.title": "JavaScript",
-  "pages.index.features.javascript.description": "Node.js provides support for the JavaScript programming language",
-  "pages.index.features.openSource.title": "Open Source",
-  "pages.index.features.openSource.description": "Node.js is open source and actively maintained by contributors all over the world",
-  "pages.index.features.everywhere.title": "Everywhere",
-  "pages.index.features.everywhere.description": "Node.js has been adapted to work in a wide variety of places"
+  "components.section.newFooter.trademark": "Trademark Policy",
```

```suggestion
  "components.sections.newFooter.trademark": "Trademark Policy",
```

Also update the key as `sections` in `NewFooter` component too (as ovflowd mentioned)